pkg/networks/commands_test.go

```diff
@@ -87,17 +84,17 @@ func TestStartCmd(t *testing.T) {
 	config, err := DefaultConfig()
 	assert.NilError(t, err)
 
-	cmd := config.StartCmd("shared", Switch)
-	assert.Equal(t, cmd, "/opt/vde/bin/vde_switch --pidfile=/private/var/run/lima/shared_switch.pid "+
-		"--sock=/private/var/run/lima/shared.ctl --group=everyone --dirmode=0770 --nostdin")
+	if _, err := exec.LookPath(config.Paths.SocketVMNet); err != nil {
+		t.Skip(err)
+	}
```

Comment:
Same thing here: just keep the old test and then duplicate the test function for the socket_vmnet use-case. Then the old test can be removed when the vde_vmnet functionality is removed.

Reply:
Added back the vde_vmnet tests